db_operations.py

```
@@ -143,16 +145,16 @@ def delete_mail(unique_id, recipient_id, bbs_nodes, interface):
             return  # Early exit if no matching mail found
         recipient_id = result[0]
-        logging.info(f"Attempting to delete mail with unique_id: {unique_id} by {recipient_id}")
+        logger.info(f"Attempting to delete mail with unique_id: {unique_id} by {recipient_id}")
         c.execute("DELETE FROM mail WHERE unique_id = ? and recipient = ?", (unique_id, recipient_id,))
         conn.commit()
         send_delete_mail_to_bbs_nodes(unique_id, bbs_nodes, interface)
-        logging.info(f"Mail with unique_id: {unique_id} deleted and sync message sent.")
+        logger.info(f"Mail with unique_id: {unique_id} deleted and sync message sent.")
     except Exception as e:
-        logging.error(f"Error deleting mail with unique_id {unique_id}: {e}")
+        logger.error(f"Error deleting mail with unique_id {unique_id}: {e}")
```

Ruff (0.15.2)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)